@here/harp-omv-datasource/test/OmvTest.ts

```diff
@@ -121,13 +121,11 @@ describe("OmvFeatureFilterDescriptionBuilder", function() {
         const filterBuilder = new OmvFeatureFilterDescriptionBuilder();
 
         filterBuilder.processLayer("road");
-        filterBuilder.processLine(
-            "road",
-            OmvGeometryType.LINESTRING,
-            "street",
-            OmvFilterString.StringMatch.Match,
-            OmvFilterString.StringMatch.Match
-        );
+        filterBuilder.processLine({
+            layer: "road",
+            geomType: OmvGeometryType.LINESTRING,
+            featureClass: "street"
```

Comment:
Is it difficult to add a test to test the matchAttribute function?

Reply:
Good idea, added thanks.